standard/enums.md

```diff
@@ -40,9 +40,9 @@ enum_body
     ;
 ```
 
-Each enum type has a corresponding integral type called the ***underlying type*** of the enum type. This underlying type shall be able to represent all the enumerator values defined in the enumeration. If the *enum_base* is present, it explicitly declares the underlying type. The underlying type shall be one of the *integral types* ([§9.3.6](types.md#936-integral-types)) other than `char`; specified either by keyword (*integral_type*), or by one of the full type names that the integral types alias ([§9.3.5](types.md#935-simple-types)) (*integral_type_name*).
+Each enum type has a corresponding integral type called the ***underlying type*** of the enum type. This underlying type shall be able to represent all the enumerator values defined in the enumeration. If the *enum_base* is present, it explicitly declares the underlying type. The underlying type shall be one of the *integral types* ([§9.3.6](types.md#936-integral-types)) other than `char`. The underlying type may be specified either by a `simple_type` ([§9.3.5](types.md#935-simple-types)), or an `integral_type_name`. The `integral_type_name` is resolved in the same way as `type_name`, including taking any using directives ([§14.5](namespaces.md#145-using-directives)) into account.
```

Comment:
Typo, replace: “specified either by a `simple_type`”
With: “specified either by an `integral_type`”

Reply:
Indeed - I'm not sure how I ended up with that :(